src/streamlink/plugins/eltrecetv.py

```diff
@@ -0,0 +1,41 @@
+import re, html
+
+from streamlink.plugin import Plugin
+from streamlink.plugin.api import http, useragents, validate
+from streamlink.stream import HLSStream, HTTPStream
+from streamlink.utils import parse_json
+
+
+class ElTreceTV(Plugin):
+    _url_re = re.compile(r'https?://(?:www\.)?eltrecetv.com.ar/.+')
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls._url_re.match(url)
+
+    def _get_streams(self):
+
+        video_url_found_hls = ""
+
+        if self.url[self.url.index('eltrecetv.com.ar/') +17:].startswith("vivo"):
+            http.headers = {'Referer': self.url,
+            'User-Agent': "ArtearPlayer/3.2.42 (Linux;Android 4.4.2) ExoPlayerLib/1.5.9"}
+            video_url_found_hls = "http://stream.eltrecetv.com.ar/live13/13tv/13tv1/playlist.m3u8"
+        else:
+            http.headers = {'Referer': self.url,
+            'User-Agent': useragents.CHROME}
+            res = http.get(self.url)
+            video_search = res.text
+            video_search = video_search[video_search.index('data-kaltura="') +14:]
+            video_search = video_search[: video_search.index('"')]
+            video_search = html.unescape(video_search)            
+            json_video_search = parse_json(video_search)
```

Comment:
you could replace `video_search` and `json_video_search` with this,
to fix the travis error and you need to remove the `import html`
```py
            _player_re = re.compile(r'''data\-kaltura="([^"]+)"''')
            match = _player_re.search(res.text)
            if not match:
                return
            json_video_search = parse_json(match.group(1).replace("&quot;", '"'))
```